src/main/java/me/jellysquid/mods/sodium/client/gl/shader/GlProgram.java

```diff
@@ -87,7 +87,7 @@ public Builder attachShader(GlShader shader) {
                 LOGGER.warn("Program link log for " + this.name + ": " + log);
             }
 
-            int result = GL20C.glGetProgrami(this.program, GL20C.GL_LINK_STATUS);
+            int result = GlStateManager.glGetShaderi(this.program, GL20C.GL_LINK_STATUS);
```

Comment:
Does `GlStateManager` not have a `glGetProgrami`? Because this seems like a mistake.
Or does it just not matter at all?

Reply:
oh, i missed that bit. ill fix that now